server/index.js

```diff
@@ -10,28 +10,75 @@ app.use(express.static(`${__dirname}/../client/public`));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-// ROUTES
-app.get('/api/reviews/:search', (req, res) => {
-  db.find(req.params.search, (err, results) => {
+// OLD FRONT END ROUTES
+app.patch('/api/reviews/:id', ({ params: { id }, body: { reviewId, field, value } }, res) => {
+  db.update(id, reviewId, field, value, (err, results) => {
     if (err) {
-      res.status(400).send(err);
+      res.sendStatus(400);
     } else {
-      res.status(200).json(results);
+      res.send(results);
     }
   });
 });
 
-app.patch('/api/reviews/:id', (req, res) => {
-  const { id } = req.params;
-  const { reviewId } = req.body;
-  const { field } = req.body;
-  const { value } = req.body;
-  db.update(id, reviewId, field, value, (err, results) => {
+app.get('/api/reviews/:id', ({ params: { id } }, res) => {
+  db.find(id, (err, results) => {
     if (err) {
-      res.status(400).send(err);
-    } else {
-      res.status(204).json(results);
+      console.log(err);
+      res.sendStatus(400);
+      return;
+    }
+    res.send(results);
+  });
+});
+
+// NEW API ENDPOINTS
+
+// CREATE
+app.post('/api/reviews/', ({ body }, res) => {
+  db.create(body, (err, results) => {
+    if (err) {
+      console.log(err);
+      res.sendStatus(500);
+      return;
+    }
+    res.send(results);
+  });
+});
+
+// READ
+app.get('/api/reviews/:id', ({ params: { id } }, res) => {
+  db.findId(id, (err, results) => {
+    if (err) {
+      console.log(err);
+      res.sendStatus(400);
+      return;
+    }
+    res.send(results);
+  });
+});
+
```

Comment:
How is this differentiated from the other GET route? Won't there be a conflict?

Reply:
all new CRUD paths are preceded with /new-api/ now to maintain front-end functionality.